static/app/components/events/interfaces/breadcrumbs/breadcrumbs.tsx

```diff
@@ -51,10 +58,47 @@ function Breadcrumbs({
   emptyMessage,
 }: Props) {
   const [scrollbarSize, setScrollbarSize] = useState(0);
+  const {projects, fetching: loadingProjects} = useProjects();
+
+  const maybeProject = !loadingProjects
+    ? projects.find(project => {
+        return event && project.id === event.projectID;
+      })
+    : null;
 
   const listRef = useRef<List>(null);
   const contentRef = useRef<HTMLDivElement>(null);
 
+  const sentryTransaction = breadcrumbs.filter(
+    crumb =>
+      crumb.breadcrumb.category === 'sentry.transaction' &&
+      defined(crumb.breadcrumb.message) &&
+      isEventId(crumb.breadcrumb.message)
+  );
+
+  const sentryTransactionIds = sentryTransaction
+    .map(crumb => crumb.breadcrumb.message)
+    .filter(defined);
+
+  const {data: transactionEvents} = useApiQuery<{
+    data: BreadcrumbTransactionEvent[];
+    meta: any;
+  }>(
+    [
+      `/organizations/${organization.slug}/events/`,
+      {
+        query: {
+          query: `id:[${sentryTransactionIds}]`,
+          field: ['title'],
+          project: [maybeProject?.id],
+        },
+      },
+    ],
+    {
+      staleTime: Infinity,
```

Comment:
We want to disable this query when there's no project or transaction Ids, right?

Reply:
yup, that got lost in moving it to the diff file. thanks for catching!